public/about.html

```diff
@@ -77,10 +80,10 @@
       </nav>
    </header>
 
-   <div class="container container-small">
+   <div class="container container-small" id="main-content">
       <h1 class="page-title hyphenate" id="lead">About BLE LTER</h1>
       <br>
-      <figure class="figure m-0">
+      <figure class="figure m-0"  role="none">
```

Comment:
why "none"?

Reply:
here's the relevant bit from the checker
![image](https://user-images.githubusercontent.com/42045942/116715963-d5afe980-a99c-11eb-9177-2c882e0b51c2.png)